sdk/core/Microsoft.ClientModel.TestFramework/src/RecordedTests/TestRecording.cs

```diff
@@ -34,6 +34,29 @@ public class TestRecording : IAsyncDisposable
     private readonly TestProxyProcess? _proxy;
     private readonly RecordedTestBase _recordedTestBase;
 
+    // This is a list of all default sanitizers applied in the test-proxy EXCEPT AZSDK0000 which is the Authorization
+    // header sanitizer, which should not be removed
+    private static List<string> _defaultSanitizersApplied = [ "AZSDK1000", "AZSDK1001", "AZSDK1002", "AZSDK1003",
```

Comment:
Test proxy only supports removing sanitizers by their specific code. Open AI can't really have all these Azure codes in their repo.
I don't love this design but I couldn't think of anything better.